### ISSUE TITLE
Scratch folder somehow making it onto the output image

### ISSUE THREAD

**Issue by GoogleCodeExporter** [n8felton/instadmg#89]
```
What steps will reproduce the problem?
1. Run instaup2date with the --instadmg-scratch-folder argument
2. Image a computer, boot
3. Navigate to /Volumes, to see a folder with the same path as the 
scratch-folder argument from the mac you built the image on

Not really sure how to troubleshoot this one... I can't find anything in the 
InstaDMG logs, or in the system.log on the mac that I've imaged. I first 
noticed this in r416.

What OS are you running on (booted from)?

OSX 10.6.7

What OS are you installing?

OSX 10.6.7 via -> 10.6.3 Retail DVD, 10.6.7 Combo updater.



Please provide any additional information below.


Basically, I've got a /Volumes/Scratch on my instadmg build machine, and tell 
instaup2date to use that filepath for scratch space, and when I boot a machine 
that I've imaged, I see a folder called 'Scratch' that's 4KBs in size, but 
appears to have all the subdirectories in the /Volumes/Scratch drive on my 
build machine.

```

Original issue reported on code.google.com by `stephenj...@gmail.com` on 27 Apr 2011 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
I should clarify that the 'Scratch' folder appears at /Volumes/Scratch on the 
output image, but only after it's been booted.
```

Original comment by `stephenj...@gmail.com` on 28 Apr 2011 at 4:15

**Comment by GoogleCodeExporter** [comment]
```
I can confirm I've seen this on one of my deployed images in the wild. When I 
have some time I'll try and do an audit of the /Volumes directory on other 
running systems. Seems this is probably a fairly rare occurrence, but odd 
nonetheless.
```

Original comment by `t...@synthist.net` on 7 Aug 2011 at 5:26

**Comment by GoogleCodeExporter** [comment]
```
I have the same issue. Is not a big deal as the folder is very small in size 
and does not contain any secure info.

I'll dig more into it
nbalonso
```

Original comment by `nbalonso` on 25 Oct 2011 at 3:00
